src/app/pages/my-request-detail/my-request-detail.component.html

```diff
@@ -55,41 +55,46 @@ <h2>Queries</h2>
     ></mdm-data-query-row>
 
     <h2>Data elements</h2>
-
     <p *ngIf="request.status === 'unsent'">
       You can add more data elements to this request by
       <a routerLink="/browse">browsing</a> or <a routerLink="/search">searching</a> our
       catalogue.
     </p>
 
+    <div class="mdm-my-request__request-elements-footer">
```

Comment:
I have renamed the CSS class to &__request-elements-selectall since it is only applied to the selectall region of the page. I have also updated the css to include an indentation so that the "select all" checkbox lines up with everything below it.